cardemo/Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM golang:1.18.3 as builder
+FROM golang:1.21.5 as builder
 RUN mkdir /build
 ADD . /build/
 WORKDIR /build
 RUN CGO_ENABLED=0 GOOS=linux go build -a -o ./cardemo ./main.go
 
 FROM alpine:3.13.6
+RUN apk update && apk add --no-cache curl
 WORKDIR /timeplus
+RUN curl -o /timeplus/routes.json https://tp-solutions.s3.us-west-2.amazonaws.com/nyc-trip-data/routes.json
```

Comment:
I am fine with this, but I don't think it's so necessary to create the `tp-solutions` S3 bucket at first place. `timeplus.io` is already a public accessible S3 bucket with anonymous web access.